README.md

```diff
@@ -52,6 +52,12 @@ text editor like _JEdit_, to a full IDE- for editing of course ;). Preferably
 something that can check XML file validity. To view the resulting PDF files a
 PDF viewer is necessary.
 
+### Building PDF's
```

Comment:
This looks like a duplicate one. I prefer not to add this, as the toolchain repository will be 'revamped'

Reply:
Hey @gronke - it looks like this is the only outstanding issue for this PR. Could you please ensure that there is only one header? IMHO the correct spelling is `Building PDFs`